content/api/cli.md

```diff
@@ -1,8 +1,264 @@
+This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.
+For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.
+Any parameters sent to the CLI will map to a corresponding parameter in the config file.
+
+## Installation
+
+``` sh
+$ npm install webpack -g
+```
+
+The `webpack` command is now available globally.
+
+### Common Usage
+
+``` sh
+webpack <entry> [<entry>] <output>
+```
+| Parameter | Configuration Mapping         | Explanation                                                                                                                                                             |
+|-----------|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| entry     | entry                         | A filename or a set of named filenames which act as the entry point to build your project. If you pass a pair in the form of `=` you can create an additional entry point. |
+| output    | output.path + output.filename | A path and filename for the bundled file to be saved in.                                                                                                                |
+
+#### Examples
+
+If your project structure is as follows -
+```
+.
+├── dist
+├── index.html
+└── src
+    ├── index.js
+    ├── index2.js
+    └── others.js
+```
+
+`webpack src/index.js dist/bundle.js`
+	This will bundle your source code with entry as `index.js` and the output bundle file will have a path of `dist` and the filename will be `bundle.js`
+
+	| Asset     | Size    | Chunks      | Chunk Names |
+	|-----------|---------|-------------|-------------|
+	| bundle.js | 1.54 kB | 0 [emitted] | index       |
+		[0] ./src/index.js 51 bytes {0} [built]
+		[1] ./src/others.js 29 bytes {0} [built]
+
+`webpack index=./src/index.js entry2=./src/index2.js dist/bundle.js`
+	This will form the bundle with both the files as separate entry points.
+
+	| Asset     | Size    | Chunks        | Chunk Names   |
+	|-----------|---------|---------------|---------------|
+	| bundle.js | 1.55 kB | 0,1 [emitted] | index, entry2 |
+	[0] ./src/index.js 51 bytes {0} [built]
+	[0] ./src/index2.js 54 bytes {1} [built]
+	[1] ./src/others.js 29 bytes {0} {1} [built]
+
+### Common options
+
+<b>List all of the options available on the cli</b>
+``` sh
+webpack --help , webpack -h
+```
+
+<b>Build source using a config file</b>
+<p>Specifies a different configuration file to pick up. Use this if you want to specify something different than `webpack.config.js`, which is the default.</p>
+
+``` sh
+webpack --config example.config.js
+```
+
+<b>Send environment variable to be used in webpack config file</b>
+``` sh
+webpack --env=DEVELOPMENT
```

Comment:
It would be good to show how to capture the variable within configuration (`module.exports = function(env) {...}`). I would also mention that it's **not** possible to pass custom flags here (disallowed in webpack 2).